libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/TestActions/AssertNoActivity.cs

```diff
@@ -58,7 +58,7 @@ public override Task ExecuteAsync(TestAdapter adapter, BotCallbackHandler callba
         {
             if (adapter.ActiveQueue.Count > 0)
             {
-                throw new Exception($"{GetConditionDescription()}");
+                throw new InvalidOperationException($"{GetConditionDescription()}");
```

Comment:
I think this should be an `ArgumentException` as the error is related to the `adapter` argument.

Reply:
fixed this one, ty